test/x509_load_cert_file_test.c

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <stdio.h>
+#include <openssl/err.h>
+#include <openssl/x509_vfy.h>
+
+#include "testutil.h"
+
+static int test_load_cert_file(int n)
+{
+    int ret = 0;
+    X509_STORE *store = NULL;
+    X509_LOOKUP *lookup = NULL;
+    STACK_OF(X509) *certs = NULL;
+    const char *chain = test_get_argument(n);
```

Comment:
This is a misuse of the `n` parameter. The argument to this function is the index of the test. The test will be repeated a given number of times and `n` tells you which iteration this is.

Reply:
Actually this is only a partial misuse - there could be a loop going through all arguments here. On the other hand this requires all the test PEM files having 4 certificates because this is hardcoded below so I am not quite sure it is that useful to loop.